palera1n.sh

```diff
@@ -158,11 +159,11 @@ _exit_handler() {
     [ $? -eq 0 ] && exit
     echo "[-] An error occurred"
 
-    cd logs
+    pushd "$rootDir/logs"
     for file in *.log; do
         mv "$file" FAIL_${file}
     done
-    cd ..
+	 popd
```

Comment:
this would lead back to the root dir, which is not necesairily where the wd of the script was, so use popd to get back where it was.

Reply:
This is probably not necesairy however, it might prevent headaches in the future.